Harvest.Core/Domain/Project.cs

```diff
@@ -85,7 +85,6 @@ public Project()
         [Display(Name = "Created By")]
         public User CreatedBy { get; set; }
 
-        [JsonIgnore]
```

Comment:
Don't think you want to remove this.

Reply:
nm I'm probably wrong.

Reply:
Is there another way to get the data if we JsonIgnore it? I JsonIgnored the Project field in Account.cs to prevent circular referencing. I think I mentioned this in #sow-what

Reply:
I think you did it correctly.